README.md

```diff
@@ -13,39 +13,54 @@ Babel plugin to generate a namespace based on filename.
 
 Directories and filenames are typically descriptive by nature, which make them great for namespacing your code.
 
-With modern JS tooling like Babel and Weback using something like `__dirname` or `__filename` can lose it’s meaning due to code bundling. The aim of this plugin is to give that meaning back, by transforming the value `__filenamespace` into a configurable static string representing the filename and path.
+The node modules [__dirname][__dirname] and [__filename][__filename] are perfect for this, but there is a problem, with modern JS tooling like Babel and Weback these modules lose meaning due to code bundling.
```

Comment:
Maybe update

`but there is a problem, with modern JS tooling like Babel and Weback these modules lose meaning due to code bundling.`

to 

`but there is a problem, with code bundling these modules lose meaning. The purpose of this plugin is to give that meaning back.`